README.md

```diff
@@ -60,6 +60,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [Perceval](https://github.com/Quandela/Perceval) - [Quandela](https://www.quandela.com)'s software library for programming realistic photonic quantum computers.
 - [ProjectQ](https://github.com/ProjectQ-Framework/ProjectQ) - Hardware-agnostic framework with compiler and simulator with emulation capabilities.
 - [PyQudit](https://github.com/Ordoptimus/pyqudit) - Python package for generalized and universal versions of quantum gates in N-dimensions.
+- [pytket](https://docs.quantinuum.com/tket/) - Quantum computing toolkit for building, compiling and executing quantum circuits (developed by Quantinuum).
```

Comment:
I'm surprised that this wasn't on the list already. (I'm also surprised that entries involving Quantinuum aren't using the Oxford comma.)

Reply:
Haha fixed thanks. I work in Cambridge ;)